### ISSUE TITLE
library should not depend on external resources

### ISSUE THREAD

**Issue by edrevo** [eclipsesource/play-json-schema-validator#129]
right now http://json-schema.org is down (see https://github.com/json-schema-org/json-schema-org.github.io/issues/213). All validations in this library are failing with messages similar to this:

```
Could not resolve ref http://json-schema.org/draft-04/schema#/properties/type
```

Meanwhile, projects that use other json schema validators like https://github.com/java-json-tools/json-schema-validator aren't failing because they have the json-schema schema as a resource in the library.

This library should follow the same mechanism, to avoid depending on external URLs which might or might not exist in the future.

**Comment by edgarmueller** [comment]
Indeed, thanks for reporting, I'll try to address it today.

**Comment by edgarmueller** [comment]
This should be fixed with [0.9.5-M2](https://github.com/eclipsesource/play-json-schema-validator/releases/tag/v0.9.5-M2). Note that you need to explicitly pass the desired schema version to the validator, as in:
```Scala
val validator = SchemaValidator(Some(Version7))
```
Please re-open if you experience any problems re-lated to this issue.
